README.md

```diff
@@ -316,6 +361,27 @@ environment variable on `1` before executing the test script:
 export DEBUG=1
 ```
 
+<a name="vagrant-spec-based-tests"></a>
+#### Vagrant Spec based tests
+
+The source contains also a set of [vagrant-spec](https://github.com/mitchellh/vagrant-spec) acceptance tests. They can be run via:
+
+```
+vagrant-spec test --components provider/virtualbox/cdk
+vagrant-spec test --components provider/libvirt/cdk
+```
+
+In order to successfully run the tests, you need to set the following
+environment variables:
+
+* _VAGRANT_SPEC_BOX_VIRTUAL_BOX_ - URL to the VirtualBox box for testing
```

Comment:
You already can use a file path. Just use (file://<path to the box file). That's how I do it as well and how it is intended. Downloading the box on each test would take so long, especially since the box won't be cached.

As a dev you just download the boxes you are interested to your preferred location and point the variables to it. Then you can run the tests against these boxes as often as you like. I envision this to be used in a CI environment as well. The build setup would copy the boxes to a dedicated location (probably within the checkout) and then just set the appropriate environment variables.